dist/images/ovnkube.sh

```diff
@@ -118,8 +118,8 @@ mtu=${OVN_MTU:-1400}
 ovn_kubernetes_namespace=${OVN_KUBERNETES_NAMESPACE:-ovn-kubernetes}
 
 # host on which ovnkube-db POD is running and this POD contains both
-# OVN NB and SB DB running in their own container
-ovn_db_host=""
+# OVN NB and SB DB running in their own container. Ignore IPs in loopback range (127.0.0.0/8)
+ovn_db_host=$(getent ahostsv4 $(hostname) | grep -v "^127\." | head -1 | awk '{ print $1 }')
```

Comment:
looks like we should file a bug to fix this script for IPv6 ...

Reply:
yes, though note that we don't use this script in OpenShift

Reply:
Filed #977. I can take a look at fixing that in a follow up patch set, but for now this fix works for ipv4 in my test setup.